poppy/instrument.py

```diff
@@ -976,9 +976,9 @@ def _get_weights(self, source=None, nlambda=None, monochromatic=None, verbose=Fa
             wavelengths = wavelengths * units.Unit(waveunit)
             lrange = wavelengths[throughputs > 0.4].to_value(units.m)  # convert from Angstroms to Meters
             # get evenly spaced points within the range of allowed lambdas, centered on each bin
-            lambd = np.linspace(np.min(lrange), np.max(lrange), nlambda, endpoint=False) + (
-                    np.max(lrange) - np.min(lrange)) / (2 * nlambda)
+            points = np.linspace(np.min(lrange), np.max(lrange), nlambda, endpoint=False) + (
+                     np.max(lrange) - np.min(lrange)) / (2 * nlambda)
             filter_fn = scipy.interpolate.interp1d(wavelengths.to_value(units.m), throughputs, kind='cubic',
                                                    bounds_error=False)
-            weights = filter_fn(lambd)
-            return lambd, weights
+            weights = filter_fn(points)
+            return points, weights
```

Comment:
This is a significant divergence. I'm happy to drop it. But, it seems like `points` more accurately describes what the object is.